app/services/foods_service.rb

```diff
@@ -0,0 +1,20 @@
+class FoodsService
+  class << self
+    def search_by_food_db(food_input)
+      response =  conn.get("fdc/v1/foods/search?query=#{food_input}")
```

Comment:
This class is excellent, the only thing I would change is I would set the `query` param in a block the same way that you set the API key down in the `conn` method.